src/network.js

```diff
@@ -2,11 +2,11 @@ const artifact = require("@actions/artifact");
 
 export function uploadArtifact(s0) {
   const artifactClient = artifact.create();
-  const artifactName = "apk-metric-artifact";
-  const files = [`apk-metric.json`];
+  const artifactName = "metric-artifact";
+  const files = [`apk-metric.json`, `bundle-metric.json`];
```

Comment:
Keep a single file with the following schema. 
`{  master_size: --,  bundle_size: -- }`